### DISCUSSION TITLE
How to hide annoation popup / "sticky note"- icon?

### DISCUSSION THREAD

**Discussion by stefaneidelloth** [pymupdf/PyMuPDF#3244] [Q&A]
I created an annoation with a hover text:

```
vertices = [(100, 400), (200, 400), (150, 500)]

# Create a polygon annotation on the page
annotation = page.add_polygon_annot(vertices)

# Set the annotation's properties (optional)
annotation.set_border(width=1.5)
annotation.set_opacity(0.5)


popup_rect = fitz.Rect(300, 400, 500, 500)  
annotation.set_popup(popup_rect)
annotation.set_info(None, 'content', 'title')
```

![image](https://github.com/pymupdf/PyMuPDF/assets/12675339/331d6a3d-6c42-49a4-827c-ff2451f66ba5)

=> How can I hide the yellow annotation icon?  
![image](https://github.com/pymupdf/PyMuPDF/assets/12675339/74ff286c-1e15-4a09-b89e-6e8b7137b076)

**Comment by stefaneidelloth** [comment]
Disabling
`Edit => Preferences => Enable text indicators and tooltips`
in Acrobat Reader removed the sticky note icons for me. 

However, I would like to store this this info with the **document**. 

* If that is possible somehow, please let me know. 

* If there is an **alternative approach** to show hover info, that is not based on comments, please let me know, too. 
Unfortunately, the annoations and shapes do not seem to support mouse hover action listeners to be able to attach some custom JavaScript (?)


Related:
https://community.adobe.com/t5/acrobat-discussions/remove-unnecessary-comment-icon-on-highlight-comment/td-p/3530334

**Comment by JorjMcKie** [comment]
Already implemented!

If an annotation has a popup, you can request setting it "open" or "closed" - see documentation [here](https://pymupdf.readthedocs.io/en/latest/annot.html#Annot.set_open).
Independent from that, use the annotation flags to set desired bits.  in your case, you could use "hidden" 
![image](https://github.com/pymupdf/PyMuPDF/assets/8290722/78e13292-a9bc-4dc8-ba65-40b3f452d0d9)

**Reply by fhg-isi** [reply]
Thank you for the suggestion. The flags seem to be related to the whole annotation and not the extra icon that denotes that there is some annotation.

**Reply by JorjMcKie** [reply]
Got you now.
The little yellow icon seems to be a peculiarity of Adobe Acrobat.
Even if there is no popup defined, information stored in the info dictionary is displayed on hover by almost all viewers I am using: Acrobat, PDF XChange, Nitro, Foxit evince.
If no popup is defined then all the others (non-Adobe) display no yellow presence-icon.
Adobe seems to always display that if something is contained in info.
So I am afraid this is a global Adobe setting?
Anyway, I see no way to influence this behavior ...

**Comment by JorjMcKie** [comment]
Moving this to "Discussions" for potential further questions.
